puppet/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM phusion/baseimage:0.9.18
+
+MAINTAINER  eXo Platform <docker@exoplatform.com>
+
+# Set the locale
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# eXo CI User
+ENV EXO_CI_USER ciagent
+ENV EXO_GROUP ciagent
+ENV HOME /home/${EXO_CI_USER}
+
+# Required directories for ciagent
+ENV EXO_CI_BASE       /opt/ciagent
```

Comment:
We should update all CI dockerfiles with /srv/ciagent in order to be consistent with all Jenkins agents